src/Core/Models/Mail/OrganizationUserInvitedViewModel.cs

```diff
@@ -8,6 +10,8 @@ public class OrganizationUserInvitedViewModel : BaseMailModel
         public string Email { get; set; }
         public string OrganizationNameUrlEncoded { get; set; }
         public string Token { get; set; }
+        private DateTime _expirationDate => DateTime.UtcNow.AddDays(5);
```

Comment:
So instead of making this a private _property_, it should probably be relegated to a `private readonly DateTime` and be set using a normal `=` vs. an arrow property function. Bonus points for the expiration date `DateTime` value being set by the caller itself (so the time is actually accurate based on when the token value was generated from the caller).

Reply:
Changed to the solution that promised bonus points

Reply:
Bonus points achieved!